### ISSUE TITLE
Sidebar a11y violations

### ISSUE THREAD

**Issue by hxltrhuxze** [Royal-Navy/design-system#1102]
- [ ] Ensures buttons have discernible text

**Comment by hxltrhuxze** [comment]
Closing this issue as it will be superseded by the sidebar redesign.
